### DISCUSSION TITLE
subscription

### DISCUSSION THREAD

**Discussion by barayaa** [community/community#46831] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I just renewed my subscription to copilot but I still can't use the extension on the pretext that my subscription has expired

**Comment by davecheney** [comment, marked as answer]
@barayaa it looks like you card was declined.

**Reply by barayaa** [reply]
Alright thanks for your feedback! however I reassured myself to recharge my card before wanting to take my subscription how can I solve this problem please

**Reply by davecheney** [reply]
Please contact support, https://support.github.com/contact
